index.html

```diff
@@ -0,0 +1,412 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://use.fontawesome.com/1fb0eb3889.js"></script>
+    <title>Home | Revera</title>
+    <link href="css/reset.css" type="text/css" rel="stylesheet">
+    <link href="css/bootstrap.min.css" type="text/css" rel="stylesheet">
```

Comment:
You should only need one bootstrap css link, either as CDN or locally, not both :)